pin.go

```diff
@@ -1,10 +1,18 @@
 package drivers
 
-// Pin is a digital pin interface. It is notably implemented by the
-// machine.Pin type.
+// Pin is a digital pin interface.
+// It is notably implemented by the machine.Pin type.
 type Pin interface {
+	PinIn
+	PinOut
+}
+
+type PinIn interface {
 	Get() bool
-	High()
-	Low()
+}
+
+type PinOut interface {
+	High() // deprecated: use Set(true)
+	Low()  // deprecated: use Set(false)
```

Comment:
I'd prefer we do not have these in the interface and sweep and eradicate call to them everywhere in drivers.